src/platform/mix.library/ViewModels/MixDatabaseViewModel.cs

```diff
@@ -40,6 +39,18 @@ public MixDatabaseViewModel(MixDatabase entity, UnitOfWorkInfo uowInfo = null)
         #endregion
 
         #region Overrides
+
+        public override async Task Validate(CancellationToken cancellationToken)
+        {
+            if (await Context.MixDatabase.AnyAsync(p => p.SystemName == SystemName, cancellationToken))
+            {
+                IsValid = false;
+                Errors.Add(new ValidationResult("Database Existed"));
```

Comment:
@truongtphat  Only invalid when  different Id but the same name  or the update function will not work
await Context.MixDatabase.AnyAsync(p => p.Id != this.Id && p.SystemName == SystemName, cancellationToken)

Reply:
I'm going to remove this first and find another solution to validate this view model

Reply:
@nhathoang989 I updated.